### PR TITLE
Option to disable suggestions

### PR DESCRIPTION
**Pull request by simonwjackson**
more info @ #514

### CODE REVIEW

**Comment by simonwjackson** [comment]
@brookhong Any thoughts on this?

File: pages/omnibar.js [hunk 1]
```diff
@@ -995,9 +995,13 @@ var SearchEngine = (function() {
         return this.activeTab;
     };
     self.onInput = function() {
-        if (!self.suggestionURL || typeof(self.listSuggestion) !== "function") {
-            return;
-        }
+        const showSuggestions = (
+            !self.suggestionURL
```
Comment (brookhong) [inline pages/omnibar.js]: Looks like the expression here is wrong, please make sure test done.

Comment (simonwjackson) [reply pages/omnibar.js]: Can you provide more detail?
I have tested this on my machine and it is working correctly.

Comment (brookhong) [reply pages/omnibar.js]: Should it be 

    const showSuggestions = (
                self.suggestionURL
                && typeof(self.listSuggestion) == "function")
                && runtime.conf.omnibarSuggestion
    

With your change, when `omnibarSuggestion` is true(and I prefer it default to be true, as most users are used to this behavior.), no suggestion will be listed, please notice that you're using `!showSuggestions` below.

Comment (simonwjackson) [reply pages/omnibar.js]: You're right. Thanks for your input! Sending the pull request now.

**Comment by simonwjackson** [comment]
@brookhong Logic should be OK now

File: pages/omnibar.js [hunk 2]
```diff
@@ -995,9 +995,11 @@ var SearchEngine = (function() {
         return this.activeTab;
     };
     self.onInput = function() {
-        if (!self.suggestionURL || typeof(self.listSuggestion) !== "function") {
-            return;
-        }
+        const canSuggest = self.suggestionURL && typeof(self.listSuggestion) === "function"
```
Comment (brookhong) [inline pages/omnibar.js]: Great, thanks for the fix. 👍 

Could you please also add semicolon at end of these two lines?

**Comment by simonwjackson** [comment]
@brookhong semicolons added

**Pull request merged**